lib/vtls/schannel.c

```diff
@@ -1838,15 +1851,16 @@ schannel_recv(struct connectdata *conn, int sockindex,
     }
   }
 
-  infof(data, "schannel: encrypted data buffer: offset %zu length %zu\n",
-        BACKEND->encdata_offset, BACKEND->encdata_length);
+  DEBUGF(infof(data,
+               "schannel: encrypted data buffer: offset %zu length %zu\n",
+               BACKEND->encdata_offset, BACKEND->encdata_length));
 
-  infof(data, "schannel: decrypted data buffer: offset %zu length %zu\n",
-        BACKEND->decdata_offset, BACKEND->decdata_length);
+  DEBUGF(infof(data,
+               "schannel: decrypted data buffer: offset %zu length %zu\n",
+               BACKEND->decdata_offset, BACKEND->decdata_length));
 
 cleanup:
   /* Warning- there is no guarantee the encdata state is valid at this point */
-  infof(data, "schannel: schannel_recv cleanup\n");
```

Comment:
Can you please DEBUGF wrap these removed lines instead? This is still useful information.

Reply:
Really? Ok!

Reply:
I would also prefer to keep all infof's with DEBUGF and not remove any of them. They were and are probably really useful for state machine and byte count tracking since these were very hard things to work out during the initial development of this backend.

Reply:
Well sure, but some of them are more or less ridiculous to have there all the time (which is why I think I kept two or three removals). They're very easy to add there again if you really need for debugging...